src/steamship/client/client.py

```diff
@@ -156,7 +198,7 @@ def list_comments(
         external_id: str = None,
         external_type: str = None,
         external_group: str = None,
-    ) -> Response[TaskCommentList]:
+    ) -> IResponse[TaskCommentList]:
```

Comment:
@EniasCailliau I'm not sure if my branch is out of sync, but Pycharm was complaining here so I altered the response type on this

Reply:
True. I never fix these since we have so many MyPy issues in the codebase. Holding off until we start "the big refactor" once we plan to go public.

Reply:
👍